src/visualization.jl

```diff
@@ -196,9 +186,10 @@ end
 
         for (i, (name, integral)) in enumerate(pairs(ints))
             name in exclude && continue
+            name_function = getfield(@__MODULE__, name)
```

Comment:
After thinking more about it (or better not thinking about it), I noticed that this can be problematic if you define a function for an integral outside of the DispersiveShallowWater.jl module. Wold it make sense to just store/use the functions here?

Reply:
Good point. I didn't think about that. What exactly do mean by " just store/use the functions here"?

Reply:
The functions determining the analysis integrals, e.g., the ones that can be passed in `extra_analysis_integrals`

Reply:
Let's continue the discussion in #152, where I made a suggestion.